go-concourse/concourse/team.go

```diff
@@ -12,6 +12,7 @@ import (
 type Team interface {
 	Name() string
 
+	Config(teamName string) (atc.Team, bool, error)
```

Comment:
Seems like this should be called `Team` instead, since that's what we do for `.Pipeline/.Job`. :thinking: 

It's a *little* weird because it's already on a `Team` object, but it's at least consistent with the other methods, and kind of makes sense since you're trying to use team A as an access check when fetching team B.

Reply:
> It's a little weird because it's already on a Team object, but it's at least consistent with the other methods, and kind of makes sense since you're trying to use team A as an access check when fetching team B.

I think so.
I will change `Config` to `Team` :+1: